python_modules/dagster/README.md

```diff
@@ -26,55 +16,59 @@
   <a target="_blank" href="https://pypi.org/project/dagster/" style="background:none">
     <img src="https://img.shields.io/pypi/pyversions/dagster?labelColor=4F43DD&color=163B36">
   </a>
-</p>
-
-# Dagster
+  <a target="_blank" href="https://twitter.com/dagster" style="background:none">
+    <img src="https://img.shields.io/badge/twitter-dagster-blue.svg?labelColor=4F43DD&color=163B36&logo=twitter" />
+  </a>
+  <a target="_blank" href="https://dagster.io/slack" style="background:none">
+    <img src="https://img.shields.io/badge/slack-dagster-blue.svg?labelColor=4F43DD&color=163B36&logo=slack" />
+  </a>
+  <a target="_blank" href="https://linkedin.com/showcase/dagster" style="background:none">
+    <img src="https://img.shields.io/badge/linkedin-dagster-blue.svg?labelColor=4F43DD&color=163B36&logo=linkedin" />
+  </a>
+</div>
 
-Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+__Dagster is a cloud-native orchestrator for the whole development lifecycle, with integrated lineage and observability, a declarative programming model, and best-in-class testability.__
```

Comment:
Is it a little weird to have "Dagster is a" twice in a row?

Reply:
Agreed - I will fix that.